src/einsteinpy/utils/exceptions.py

```diff
@@ -10,5 +34,12 @@ def __str__(self):
 
 
 class CoordinateError(BaseError):
+    """
+    Error class for invalid coordinate operations.
+    Raised if Metric objects are defined with coordinate system \
+    not supported by geometry.
+
```

Comment:
Again, period / full-stops not required. And, the second line isn't important either. `CoordinateError` may be used for other purposes too.